js/ui/components/networkAgent.js

```diff
@@ -38,6 +39,9 @@ const NetworkSecretDialog = new Lang.Class({
         else
             this._content = this._getContent();
 
+        this._inputSourceManager = InputSourceManager.getInputSourceManager();
+        this._inputSourceManager.passwordModeEnabled = true;
```

Comment:
Are we not packing the switching indicator here?

Reply:
That one has been added to the network dialog.